packages/components/pie-icon-button/src/defs.ts

```diff
@@ -1,4 +1,9 @@
 export interface IconButtonProps {
+    /**
+     * the size of the icon button.
+     * @default medium
+     */
```

Comment:
NIT: can we remove these JSDoc comments? They aren't really helping anyone and they're quite messy

Reply:
I added them as that was the current convention in the codebase tbh.

In defence of them though, they do tell you what the default is, just from looking at the `defs` file (and I can see that for more complex props, the descriptions could become useful.

Reply:
I would argue that the default value isn't really a property of the interface, it's a property of the button. I'd also be in favour of removing these except in the cases where they do help with understanding what the prop is used for

Reply:
As a rule of thumb I'd give every convention a second thought at the moment as a lot of it is a result of proof of concept mode and might not be appropriate going forward

Reply:
> As a rule of thumb I'd give every convention a second thought at the moment as a lot of it is a result of proof of concept mode and might not be appropriate going forward

Completely agree. I won't make any changes as part of this PR, as it's out of scope. Best thing to do (imo) would be to bring this up as a discussion after standup, to make sure everyone is aware of the discussion, and then if agreed upon we can raise a PR to remove these.